contracts/examples/AyiiProduct.sol

```diff
@@ -395,35 +395,76 @@ contract AyiiProduct is
 
     function processPoliciesForRisk(bytes32 riskId, uint256 batchSize)
         external
+        onlyRole(INSURER_ROLE)
```

Comment:
or product owner?

Reply:
by intention: the product owner role is only handling the permissions for interaction between the product and the instance. if the product itself wants to go for a permissioned setup with its own stakeholder it is cleaner to create a dedicted role for that.